### PR TITLE
Project BookStore: Milestone 2

### PR DESCRIPTION
**Pull request by diazgio**
### in this PR we:
 - Create components, actions, containers, reducers folders.
 - Create book component file, actions CREATE and REMOVE file.
 - Add redux syntax to manage states from components.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import { createStore } from 'redux';
+import { Provider } from 'react-redux';
 import App from './App';
```
Comment (arslanbisharat) [inline src/index.js]: - [x] index.js should import the `App `component from the **components** folder. 
`import App from './components/App';`

File: src/reducers/books.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      title: 'Red Cap',
+      category: 'Kids',
+    },
+    {
+      id: 2,
+      title: 'Second World War',
+      category: 'History',
+    },
+  ],
+};
```
Comment (arslanbisharat) [inline src/reducers/books.js:17]: - [x] This piece of code is not supposed to be here but it should be in `src/index.js`. Please move it to there.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested 💀 

There is always room to improve your code. Your project has met almost all the requirements but I would like to suggest some changes. I hope the next TSE who will review this project would be really impressed if you make those changes. 

- [x] `src/index.js` should add example books to the initial state object [screenshot from pathwright](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/bookstore/InitialBooks.png) but you have missed this. 

- [x] App component should be placed in `src/components` directory but in your project, it is placed in `src` directory. 

When you are done with the changes please request another review. 

_Don't get discouraged. There are some students who make more mistakes than you do. We are here to learn. Everyone from Micrverse is here to help you._ 

**GOOD LUCK!**

In case of any questions, feel free to reach out on slack @arslanbisharat 📞 . 

**Best, 
Arslan Bisharat**

**Review by addod19 — APPROVED** [review]
## Status: Approved :trophy: :balloon: 

Hi **TEAM**, congratulations on this milestone :fireworks: The initial state is in the src/index.js file, ie passing the initial state object to the create store method and the App component have been moved to the Components directory where all stateless components should be :1st_place_medal: Awesome
I hereby approve the project :smile: 

Kindly go ahead and merge into development, and continue the next milestone :champagne: 
Do not forget to submit the project completion form :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Pull request merged**